manifests/t/TeamSonarr/Sonarr/3.0.7.1477/TeamSonarr.Sonarr.locale.en-US.yaml

```
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.1.0 $debug=QUSU.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: TeamSonarr.Sonarr
+PackageVersion: 3.0.7.1477
```

```suggestion
PackageVersion: 3.0
```
This must match Control Panel.